sample_creation.py

```diff
@@ -0,0 +1,569 @@
+import argparse
```

Comment:
Is this script a replacement for Images_to_samples.py? If so, I don't think it's very usefull to have both.

Reply:
not necessarily, it exists to support the new way our images(*single bands) are called and processed. Let me know if you have suggestions where you want it to live.